src/app/(home)/mainPage.module.scss

```diff
@@ -0,0 +1,26 @@
+.mainPageWrap {
+	display: flex;
+	flex-direction: column;
+	gap: 62px;
+	background: var(--color-pure-light);
+	padding: 56px 80px;
+	@media (width < 1920px) {
+		& {
+			padding: 48px;
+			gap: 140px;
+		}
+	}
+
+	@media (width < 1280px) {
+		& {
+			padding: 0px 48px 36px;
+			gap: 0px;
+		}
+	}
+
+	@media (width < 768px) {
+		& {
+			padding: 0px 0px 20px;
+		}
+	}
+}
```

Comment:
setting side paddings for containers is antipattern. it's better to use _max-width: Npx_ and _width: 100%_ instead

Reply:
using percentage in width property is antipattern too